Lab2/Lab2.ipynb

```diff
@@ -522,24 +515,24 @@
M/NGYAZweNnZJKg8UQJWAHeWHEVq7WzgnMx8uPj5OS8z55UdSgL2A67KzPrMXAj8DfBhh3pdZl6bmddTWU2kpQ8D4zPz6sysB84CXh0RB3V2TwvPzh0INGXm5Bb7nsYfAupjImI3Kt+v48vOIkXEcOAc4L/LziK1FBFDgFHALhExNSLmFt0Zty47mwT8H3BiRLw4IvYE3kOl+JT6ildQqYWAyoMQYBpV1EYWnp3bFljZat9KKl1zpD4hIrYA/gBcnpkTy84jAd8DLsnMOWUHkVrZDdgC+AjwZirdGV8DfLvETNIm91L5D/hVwFwq3RivLzOQ1EqXayMLz86tAYa32jccWF1CFumfRMSLgCuojEM+veQ4EhFxGPAO4OclR5Hasr54/2VmLsjMJcDPgPeWmEna9Pv8Virj57YBdgZeQmU8stRXdLk2svDs3GRgaEQc0GLfq7E7o/qAiAjgEip/wT8+MzeWHEkCOBoYCcyOiIXAV4DjI+KJMkNJAJm5nMqTpCw7i9TKjsDewPmZuSEzlwKX4h9F1LeMp1ILAc/Ph7M/VdRGFp6dKPotXwucExHbRMSbgOOoPGGSynYRcDDw/sxc39nJUi8ZTeWX0GHF62LgJuDd5UWSXuBS4AsRsWtEvAT4InBjuZE02BVP32cAp0bE0IjYATiJFuPppN5SfA8OA4YAQyJiWLG0z3XAoRFxfHH8u8C4aoZ6WXhW5/PA1lSmt/4TcGpm+sRTpSrWTPpPKv9hvzAi1hSvj5ebTINdZq7LzIWbXlS65dRnZl3Z2aTC96gs8zMZeA54EvhBqYmkig8DxwJ1wFSgEfhSqYk0WH2bytCEM4BPFNvfLn6XH0/lZ+Zy4AjgxGpuGJn2NJEkSZIk9RyfeEqSJEmSepSFpyRJkiSpR1l4SpIkSZJ6lIWnJEmSJKlHWXhKkiRJknqUhackSZIkqUdZeEqSBpyIOCsiMiJubePYNRFxTwmxJEkatCw8JUkD2bsi4nVlh5AkabCz8JQkDVTLgHHAt8oO0p9FxBYRMaTsHJKk/s3CU5I0UCXwP8AHIuKVm3txRJweEXMiYm1EXB8Rby+67x7d4pz/jojHImJlRCyKiBsi4mWt7nNP0b33MxExIyLWRMQVEbFVRLw+Ih4t9t0TEfu0uG5k0d6JEXFpRKyKiLkR8Yni+NciYn5E1EXEjyLiRS2uPSgirizyr4uI8RHxxZbndPB1b8p7ckRMA+qBPTq7Z0QMLfKc2cY9742Ia4vtHSLiN8W59RExOyJ+vbn/+0iS+pehZQeQJKkHXQ2cTeWp54nVXhQRHwJ+CVwI/BU4CrikjVP3As4HZgHDgVOAByLiwMxc2eK8I4GdgS8A+wA/B9YDRwA/BtYC5wGjgWNbtfEj4A/A8cBngcsj4jXAvsXnw4HvA08CVxbX7AlMKq5bDRxW/DtsDZxbxT/Bm4D9ga8D64CVwIEd3TMzGyPicuDTEXFOZiZARLwUeDNwXHHvnwFvBL4ELAT2Bt5SRSZJUj8Wxe8FSZIGjIg4Czg9M3eOiE9TKRoPzszJEXENsHNmHt3B9Y8BizPzfS32XQicChyTmfe0cc0QYEtgMXBaZv6u2H8PlSJt303FaERcBZwAvDUz7yv2fR64ANgmM9dFxEhgBnBZZn6mOGc4sBSYCRyUmU3F/keBGZn5sTZyBTAE+Brw75n50k7+7e6hUhDvl5kL2zmnzXtGxAHAZOBtmXl3se8c4GRgr6I4fRb4VWb+sqMckqSBxa62kqSB7vfAbOAb1ZxcFJCHAWNaHWr9mYg4MiJuj4ilQOP/b+/uQeSqogCO/w8EkogiRLZQC7FSFOxSiYVi4QcxImohGrQQwWgq00gURCMYbPxoLDYpRASRxMJKSSQaEGUFSVSUxM4PZEHWiIbsao7FeRPfviTO091xZyf/HyzMu3Pv5c42u2fOufdS2cELqexg20wnA3oMmAcOddoALuuM3T94kZnHgVng4CDobI29vLWudRHxTEQcA04CC8BO4MqIWDP4nE157JpBW8tn3aCzz5yZeRT4EHiwGRPAFuD1zPyjmepzYHtEPBoR3d+TJGlCGXhKkiZaE/DsAu6PiCt6DJmitqLMdtoXPTf7Md8DAniEKk/dSGU813XGznWe54FfM/NUp42eY8/W1h73AvAEVbp7W7Ou5zrzf0sFjwvAQpNhHfiJM/WZEyq7fHdEXATcRJUE72m9/xjwDvA08E1EHI2I3mXQkqTVyT2ekqTzwW5gB7VncZhZKns51WnvPt8CXABszszfoA7YATYsbanL4h7glczcNWiIiNs7fTYBa1vPP7Ren20fTp85ofbVvtz0vxH4JDO/Oj1x5hywDdgWEddR5bpvRMThdj9J0mQx4ylJmniZeRJ4kTqM59Ihff+kykE3d966o/O8HjhFBakD9zIeX+qup8phgdPlw4uyipl5JDNnWj/z3Un+7ZzNvCeAN4GtwF0sznZ2+x4GtlP/j1w97ENJklavcfjjKEnS/+E14EnqRNWDQ/o+D+yNiFepvZ3XA4Ps3qA89gB1wM6eiJgGrqVKUeeWd9n/yfvA1mY/5s9UELj2n4cs65zT1Am/J/j7pF0AIuIQsA/4gsqsPkyd6vvpEtcnSRpjZjwlSeeFzPydusakT999VDnondR+xI1UUAlwvOlzBHiIOgH2XeA+qrz0F1be48BH1Cm5u6kgr881KssyZ2bOAN8DezuHKgF8TB0+9DbwFnXNzK2Z+d0S1ydJGmNepyJJUg8RsYO6D3RDU06qc4iIa4AvgZszc/+w/pKkyWeprSRJHRExRV2/8gF1RcoN1MFE0wad5xYRlwBXAc9SGdEDK7siSdK4MPCUJOlM89RhN1uAi4EfgZeAp1ZyUavAJqoM92vggbSsSpLUsNRWkiRJkjRSHi4kSZIkSRopA09JkiRJ0kgZeEqSJEmSRsrAU5IkSZI0UgaekiRJkqSR+gtMqnJXcJiXlgAAAABJRU5ErkJggg==\n",
```

Comment:
I might suggest using a bit larger N so that the tail of the Poisson distribution goes away even more in the semi-log plot? You can still easily conclude that it appears more Guassian from what you have but just a little larger N and the Poisson "tail" may be not even noticeable. :)